frontend/src/app/models/page.tsx

```
@@ -0,0 +1,128 @@
+import { BarChart, Bot, Map, Satellite, Thermometer, Wind, AlertTriangle, Database, MapPin } from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)